controllers/flinkcluster_util.go

```diff
@@ -61,6 +61,11 @@ func getJobName(clusterName string) string {
 	return clusterName + "-job"
 }
 
+// Gets GCP service account volume name
+func getGCPServiceAccountVolumeName(clusterName string) string {
+	return clusterName + "-gcp-sa"
```

Comment:
"gcp-sa-volume"?

Reply:
Actually, this function is not used, just removed it.